server/routes/contacts/create/check-answers/createContactCheckAnswersController.test.ts

```diff
@@ -16,20 +17,24 @@ const contactsService = new ContactsService(null) as jest.Mocked<ContactsService
 let app: Express
 let session: Partial<SessionData>
 const journeyId: string = uuidv4()
-const journey: CreateContactJourney = {
-  id: journeyId,
-  lastTouched: new Date(),
-  isCheckingAnswers: false,
-  names: {
-    lastName: 'last',
-    firstName: 'first',
-  },
-  dateOfBirth: {
-    isKnown: true,
-    dateOfBirth: new Date('2024-01-01'),
-  },
-}
+let journey: CreateContactJourney
 beforeEach(() => {
+  journey = {
+    id: journeyId,
+    lastTouched: new Date(),
```

Comment:
I also need to change lastTouched to not be a Date. I guess it's working because the ISO format makes the order correct alphabetically when it gets converted to a string....
To be done in a different PR

Reply:
Not sure I understand this comment? As lastTouched is still a date?

Reply:
Yes, so one of the fixes I made to was to change dateOfBirth to not be a Date because those don't serialise in the session properly. Last touched is still a date and should be changed but I didn't want to do it in this PR. It does seem to work correctly for now though...